tests/integration/components/paper-form-test.js

```diff
@@ -50,12 +50,12 @@ test('form `onSubmit` action is invoked', function(assert) {
       {{form.input value=foo onChange=(action (mut foo)) label="Foo"}}
       {{form.input value=bar onChange=(action (mut bar)) label="Bar"}}
 
-      <button onclick={{action form.onSubmit}}>Submit</button>
+      <a {{action form.onSubmit}}>Submit</a>
```

Comment:
@miguelcobain It needs to be something besides `button`, because html buttons submit forms automatically. When it's a button here, the `onSubmit` action is called twice. Once because the action is assigned directly to the button, and once because of the form's `submit` handler.